source/code/scxsystemlib/common/scxsysteminfo.cpp

```diff
@@ -406,6 +406,10 @@ namespace SCXSystemLib
             :          /* input operands                        */
                 "a" (static_cast< unsigned int >(function))
         );
+#elif defined(ppc)
+        // TODO add CPUID support for IBM PowerPC
+        (void) function;
+        (void) registers;
```

Comment:
I think this TODO item should be done prior to checkin of this code ... I'm going to mark this as another review needed due to this.